File: FizzBuzz/Python/fizzbuzz.py
"""
FizzBuzz

Written by: Ian Doarn
"""

def fizzbuzz(n):
    if n % 3 == 0 and n % 5 == 0:
        return 'FizzBuzz'
    elif n % 3 == 0:
        return 'Fizz'
    elif n % 5 == 0:
        return 'Buzz'
    else:
        return str(n)

if __name__ == '__main__':
    for n in range(1, 101):
        print(fizzbuzz(n))

File: FizzBuzz/C/Fizzbuzz.c
// Fizzbuzz.cpp
// Written by: Ian Doarn

#include "stdafx.h"

int main()
{
	for (int i = 1; i < 101; i++) 
	{
		if (i % 3 == 0 && i % 5 == 0)
			printf("FizzBuzz\n");
		else if (i % 3 == 0)
			printf("Fizz\n");
		if (i % 5 == 0)
			printf("Buzz\n");
		else
			printf("%d\n", i);
	}
    return 0;
}

